src/Native/Runtime/threadstore.cpp

```diff
@@ -58,12 +58,19 @@ PTR_Thread ThreadStore::Iterator::GetNext()
 
 #ifndef DACCESS_COMPILE
 
+#ifdef PLATFORM_UNIX
+extern "C" int __cxa_thread_atexit(void (*)(void*), void*, void *);
+extern "C" void *__dso_handle;
+#endif // PLATFORM_UNIX
 
 ThreadStore::ThreadStore() : 
     m_ThreadList(),
     m_Lock()
 {
     SaveCurrentThreadOffsetForDAC();
+#ifdef PLATFORM_UNIX 
```

Comment:
Can we move all this to PalRedhawkUnix.cpp to avoid having the PLATFORM_UNIX ifdefs? E.g. it can be done as part of PalInit.